src/dtos/Activities/Activities.create.dto.ts

```diff
@@ -0,0 +1,37 @@
+import { IsNotEmpty, IsString, IsUUID } from 'class-validator';
+
+class ActivitiesCreateDTO {
+  @IsNotEmpty()
+  @IsUUID()
+  tutorial_id: string;
+
+  @IsNotEmpty()
+  @IsString()
+  description: string;
+
+  @IsNotEmpty()
+  @IsString()
+  title: string;
+
+  @IsNotEmpty()
+  @IsString()
+  options: string[];
+
+  @IsNotEmpty()
+  @IsString()
+  answer: string;
```

Comment:
A resposta da atividade vai ser uma string? Imagino que seja um pouco mais difícil de validar a resposta, já que a pessoa teria que escrever tudo corretamente. Acho mais fácil colocarmos as respostas como alternativas e a pessoa escolhe a correta, e no final tudo é validado junto pra ver se estão corretas ou não. O que acha?

Reply:
vou fazer a alteração para int e to estudando a viabilidade de criar uma entidade questão, a outra opção para não se utilizar a entidade questão é colocar um json dentro do postgres, porém acho que quebraria a normalidade do banco o que você acha?

Reply:
Só uma dúvida.

Estamos tratando `answer` como uma `option` correta?

Reply:
sim isso mesmo @EduardoFNeto .